types/src/on_chain_config/aptos_features.rs

```diff
@@ -106,6 +107,9 @@ impl Features {
         byte_index < self.features.len() && (self.features[byte_index] & bit_mask != 0)
     }
 
+    pub fn is_remove_detailed_error_from_hash_enabled(&self) -> bool {
+        self.is_enabled(FeatureFlag::REMOVE_DETAILED_ERROR_FROM_HASH)
+    }
```

Comment:
blank line below. 

why not append to the end?

Reply:
not sure if the order matters?

Reply:
it doesn't matter (much). but the question is "why not?".

Reply:
already move to the end